airflow/www/.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This is necessary because prettier is otherwise using the default config in `.editorconfig` that will format with an indent size of 4 the json files.